packages/docusaurus-theme-search-algolia/src/theme/SearchPage/index.js

```diff
@@ -33,9 +35,10 @@ function Search() {
       themeConfig: {algolia: {appId = 'BH4D9OD16A', apiKey, indexName} = {}},
     } = {},
   } = useDocusaurusContext();
+  const {versions} = useDocsData(DEFAULT_PLUGIN_ID);
```

Comment:
Not sure about that.

Reply:
Almost good, but could fail for some users if they use docs plugin with a custom id.

If user use twice the docs plugin (like ios + android having different versioning schemes), we should rather:

- retrieve all versions, of the 2 plugin instances (`useAllDocsData()` is a map pluginId->pluginInstanceData)
- show one select per plugin instance

For unversioned docs instances (like community in our site) it's probably not worth it to display the select, but it is useful to show 1 dropdown for iOS and one dropdown for Android, as they don't necessarily have the same versions.

Also, the facet filtering in the query should be adapted to match this multi-instance docs setup.

I think this line:

```
.addFacetRefinement('version', searchVersion)
```

Should become something like:

```
.addFacetRefinement('docusaurus_tag',' 'docs-community-current')
```

---

All this may seem a bit complicated for little ROI.
I'm not sure it's really worth letting the user do a query such as "docs from iOS v1 OR docs from android v1.1".

Unless you feel like solving this problem, what I suggest temporarily is to assume there's a single version.

Unfortunately, we can't guarantee the user won't provide a pluginId, even if there is a single docs instance, so reading the default plugin id data is not safe by default.

Something like that would be safer:

```js
function useSearchVersions() {
  const allDocsData = useAllDocsData();
  const pluginIds = Object.keys(allDocsData).length;
  if (pluginIds.length === 0) {
    return [];
  } else if (allDocsData[DEFAULT_PLUGIN_ID]) {
    return allDocsData[DEFAULT_PLUGIN_ID].versions;
  } else {
    const pluginData =
      allDocsData[DEFAULT_PLUGIN_ID] ?? allDocsData[pluginIds[0]];
    return pluginData.versions;
  }
}
```